### DISCUSSION TITLE
Importing activities from other platform

### DISCUSSION THREAD

**Discussion by kartikeyaa-k** [exoplatform/exo-community#24] [Q&A]
Hi 
Currently we are using getStream.io for activities/feed. 
We would like to switch to exo platform, but we have activities on getStream platform. GetStream can gives our data in any asked format for example JSON. 

Does exo platform provide a way to import data from raw json or any other format?

**Comment by rdenarie** [comment]
Hello

In fact we have some API to create activities in a stream, but there are few problems :

- in eXo, an activities is attached to a space, and an author. I don't know if you have theses informations in your actual system
- Creating activity via API will loose the creationDate of your current activity. When an activity is create in eXo (via REST API or via java API), the creation date is automatically filled with currentDate. There is no way to 'pre-set' a date at activity creation

In fact, activities in a stream are by definition 'short-life' data. 
So regarding the work needed to migrate activities from a system to a new one, is it worth it ? Perhaps you should consider the option the start with fresh stream.
